src/main/resources/index.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <div>
-  A plugin that integrates with <a href="https://www.grammatech.com/products/codesonar">GrammaTech CodeSonar</a>.
+  A plugin that integrates with the <a href="https://links.codesonar.com/product-home/">CodeSonar</a> static analyzer.
 </div>
```

Comment:
This text shows up in the Jenkins UI under the Jenkins plugin management screen.

Reply:
ok